### PR TITLE
RHDEVDOCS-4642-distributed-tracing-2.7-RN

### PR DESCRIPTION
**Pull request by bburt-rh**
Summary: This PR adds release notes content for the distributed tracing 2.7 release.

- Aligned team: DevTools
- For branches: 4.9+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-4642
- Direct link to doc preview: 
- - https://52818--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html#new-features-and-enhancements-red-hat-openshift-distributed-tracing-2-7
- - https://52818--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html#distr-tracing-rn-fixed-issues_distributed-tracing-release-notes
- - https://52818--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html#distr-tracing-rn-known-issues_distributed-tracing-release-notes
- SME review: @pavolloffay @rubenvp8510 @iblancasa 
- QE review: @jkandasa 
- Peer review: @ tbd

### CODE REVIEW

**Comment by bburt-rh** [comment]
@jkandasa Can you please review this PR and provide feedback? Thanks!

**Comment by jkandasa** [comment]
including @iblancasa

File: modules/distr-tracing-rn-fixed-issues.adoc [hunk 1]
```diff
@@ -14,7 +14,21 @@ Fix - What did we change to fix the problem?
...
 ////
 
-* link:https://issues.redhat.com/browse/TRACING-2337[TRACING-2337] Jaeger is logging a repetitive warning message in the Jaeger logs similar to the following:
+* link:https://issues.redhat.com/browse/OSSM-1910[OSSM-1910]
+Because of an issue introduced in version 2.6, TLS connections could not be established with {product-title} {SMProductShortName}. 
+This update resolves the issue by changing the service port names to match conventions used by {product-title} {SMProductShortName} and Istio. 
+
+
+* link:https://issues.redhat.com/browse/OBSDA-208[OBSDA-208]
+ Before this update, the default 200m CPU and 256Mi memory resource limits  could cause {OTELShortName} to get reinstalled on large clusters.
```
Comment (pavolloffay) [inline modules/distr-tracing-rn-fixed-issues.adoc:23]: to get continuously restarted

Comment (bburt-rh) [reply modules/distr-tracing-rn-fixed-issues.adoc]: Revised per you suggestion to read, `Before this update, the default 200m CPU and 256Mi memory resource limits could cause {OTELShortName} to restart continuously on large clusters.`

**Review by iblancasa — APPROVED** [review]

**Comment by rubenvp8510** [comment]
I think we might want to include this one: https://issues.redhat.com/browse/OBSDA-220

**Review by jkandasa — APPROVED** [review]
LGTM

**Review by rubenvp8510 — APPROVED** [review]

**Comment by bburt-rh** [comment]
> I think we might want to include this one: https://issues.redhat.com/browse/OBSDA-220

I added OBSDA-220 to the Known Issues section.

**Comment by rubenvp8510** [comment]
LGTM

**Comment by bburt-rh** [comment]
@jkandasa Can you PTAL at the change I made to this PR (added a known issue) and approve again if it still looks good to you?  Thanks! https://github.com/openshift/openshift-docs/pull/52818#issuecomment-1319002659

**Review by jkandasa — APPROVED** [review]
LGTM

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**